kratos/utilities/os_utilities.cpp

```diff
@@ -41,5 +43,42 @@ std::string GetCurrentWorkingDir()
     KRATOS_CATCH("")
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
+void RemoveOnCurrentWorkingDir(const std::string& rFileName)
+{
+#ifdef KRATOS_COMPILED_IN_WINDOWS
+	if (IsDirExist(rFileName)) // It is a directory
+		RemoveDirectoryA((GetCurrentWorkingDir() + "\\" + rFileName).c_str());
+	else  // It is a file
+		remove((GetCurrentWorkingDir() + "\\" + rFileName).c_str());
+#else
+    remove((GetCurrentWorkingDir() + "/" + rFileName).c_str());
+#endif
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+bool IsDirExist(const std::string& rFolderName)
```

Comment:
```suggestion
bool DirExists(const std::string& rFolderName)
```

~~~c
if (DirExists(...)) { ... }
~~~

is easier to understand I think

Reply:
If I change this it will be changing all, otherwise many conflicts